source/common/upstream/cluster_manager_impl.cc

```diff
@@ -170,21 +170,24 @@ ClusterManagerImpl::ClusterManagerImpl(const envoy::config::bootstrap::v2::Boots
                                        const LocalInfo::LocalInfo& local_info,
                                        AccessLog::AccessLogManager& log_manager,
                                        Event::Dispatcher& main_thread_dispatcher,
-                                       Server::Admin& admin)
+                                       Server::Admin& admin, SystemTimeSource& system_time_source,
+                                       MonotonicTimeSource& monotonic_time_source)
```

Comment:
Might be nicer to combined system/monotonic into one super container struct.

Reply:
I think this is the only place both sources are needed. Don't want to build a struct for just this one case.